client/galaxy/style/scss/ui.scss

```diff
@@ -24,19 +24,18 @@
     transform: rotate(45deg);
 }
 
+// utility class from: https://github.com/twbs/bootstrap/pull/27268
+.overflow-auto {
```

Comment:
do we have to redefine the class given it was implemented in the PR mentioned above? https://github.com/twbs/bootstrap/pull/27268/files#diff-afd6c72eb64ef2c8a2d963958d6991d1R4

Reply:
ah, it is unreleased yet, so I assume the plan is to remove it when they release

Reply:
Its not in our version yet. It will be available with the next BS4 revision.